src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors.mdx

```diff
@@ -18,50 +18,92 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-Scripted browsers emulate a custom user experience by scripting browsers that navigate your website, take specific actions, and ensure specific elements are present. To create a scripted browser, go to <DoNotTranslate>**[one.newrelic.com > Synthetic monitoring > Create a monitor](https://one.newrelic.com/synthetics-nerdlets/monitor-create)**</DoNotTranslate>, then select the <DoNotTranslate>**User flow / functionality**</DoNotTranslate> tile. 
+import syntheticCreate from 'images/synthetic_screenshot-crop_create-monitor.webp'
 
-See [Scripted browser reference](/docs/synthetics/synthetic-monitoring/scripting-monitors/synthetic-scripted-browser-reference-monitor-versions-chrome-100/) for a list of all available functions.
+import syntheticUserFlowFunctionality from 'images/synthetic_screenshot-full_user-flow-functionality.webp'
 
-Scripted monitors are driven by [Selenium WebDriver](https://www.selenium.dev/documentation/webdriver/). Each time your script runs, New Relic creates a fully virtualized Selenium-driven Google Chrome browser that navigates your website and follows each script step. Synthetic monitoring includes an IDE-style script editor that suggests functions, locators, and other elements to simplify scripting.
+import syntheticConfigureMonitor from 'images/synthetic_screenshot-crop_configure-monitor.webp'
 
-For a look at how to use the Selenium IDE to record your workflows and export them in the format required by New Relic, watch this short YouTube video (3:49 minutes).
+import syntheticSelectLocations from 'images/synthetic_screenshot-crop_select-locations.webp'
+
+
+Scripted browsers emulate a custom user experience by scripting browsers that navigate your website, take specific actions, and ensure specific elements are present. [Selenium WebDriver](https://www.selenium.dev/documentation/webdriver/) drives scripted monitors. Each time your script runs, New Relic creates a fully virtualized Selenium-driven Google Chrome browser that navigates your website and follows each script step. Synthetic monitoring includes an IDE-style script editor that suggests functions, locators, and other elements to simplify scripting. See [Scripted browser reference](/docs/synthetics/synthetic-monitoring/scripting-monitors/synthetic-scripted-browser-reference-monitor-versions-chrome-100/) for a list of all available functions.
+
+## How to create a scripted browser [#create-scripted-browser]
```

Comment:
Should be "How to create a scripted browser monitor"

Reply:
Changed!